### PR TITLE
New: Toggle Menu

### PR DESCRIPTION
**Pull request by joshcox**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -158,6 +158,11 @@
         "color-convert": "1.9.1"
       }
     },
+    "any-observable": {
+      "version": "0.2.0",
+      "resolved": "https://registry.npmjs.org/any-observable/-/any-observable-0.2.0.tgz",
+      "integrity": "sha1-xnhwBYADV5AJCD9UrAq6+1wz0kI="
+    },
```
Comment (mjayjones13) [inline package-lock.json:165]: What are all these changes?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="6512ec66-1172-4fa5-996a-1c8e56f3767e" name="Default" comment="" />
+    <list default="true" id="6512ec66-1172-4fa5-996a-1c8e56f3767e" name="Default" comment="">
+      <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" afterPath="$PROJECT_DIR$/.idea/workspace.xml" />
+    </list>
```
Comment (mjayjones13) [inline .idea/workspace.xml:6]: Were all these xml changes brought in automatically with the packages that you used for the toggle menu?

File: package-lock.json (hunk 1 above)
Comment (joshcox) [reply package-lock.json:165]: Those are those ide files that I'm waiting to remove. Don't worry about them; I'm going to push up a commit afterwards to get rid of them. I just didn't want all those removals further obfuscating the PR.

File: .idea/workspace.xml (hunk 2 above)
Comment (joshcox) [reply .idea/workspace.xml:6]: ^ same

**Pull request merged**